user-guide/content/docs/architecture.md

```diff
@@ -7,13 +7,17 @@ title: Architecture
 
 {{< mermaid >}}
 erDiagram
-  CARDANO-NODE ||--o{ CARDANO-WALLET : sends-blocks-and-receives-txs
-  CARDANO-NODE ||--o{ CARDANO-DB-SYNC : sends-blocks
-  CARDANO-NODE ||--o{ CARDANO-SUBMIT-API : receives-txs
+  CARDANO-NODE ||--|{ CARDANO-WALLET : depends-on
+  CARDANO-NODE ||--|{ CARDANO-DB-SYNC : depends-on
 
-  CARDANO-DB-SYNC ||--|| POSTGRESQL : dumps-into
```

Comment:
Where did PostgreSQL go?
It's still very much a part of the stack.

Reply:
It is, but is it relevant to highlight on the high level diagram? For example, we don't highlight SQLITe for the wallet, or any dependencies for other components...

Reply:
The PostgreSQL database is special. Many users, myself included, want to query PostgreSQL directly. The PostgreSQL database schema is the public API of cardano-dbsync. Neither of these characteristics apply to the wallet's SQLite db.